app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -445,6 +490,8 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       $scope.emsCommonModel.metrics_api_port = "5432";
       $scope.emsCommonModel.default_api_port = $scope.getDefaultApiPort($scope.emsCommonModel.emstype);
       $scope.emsCommonModel.metrics_database_name = "ovirt_engine_history";
+    } else if ($scope.emsCommonModel.emstype === 'kubevirt') {
```

Comment:
Should this be `$scope.emsCommonModel.virtualization_selection === 'kubevirt'`?

since for a Container Provider, `emstype` is either 'kubernetes' or 'openshift' and I believe `virtualization_selection` is `kubevirt`

Reply:
If the above is true, can you check all `emstype/kubevirt` references below and change them appropriately?

Reply:
@AparnaKarve I think this should be completed removed. I can't think of a scenario in which we'll allow modifying the type of the 'kubevirt' provider or when we'll allow the user to add that provider explicitly and not via the openshift/kubernetes provider.